Fatal error: exception Failure("Variable with name a exists.")
